addons/space_filter.js

```diff
@@ -0,0 +1,15 @@
+module.exports.message = function(client, msg) {
+	const space_channel = "603336862878138408";
+	if(msg.channel.id != space_channel) return;
+	const split_msg = msg.content.split(" ");
+
+	split_msg.forEach(element => {
+		const filtered = element.replace(" ", "");
+
+		if(filtered.length > 1) {
+			if(!filtered.toLowerCase().startsWith("https://") && !filtered.toLowerCase().startsWith("http://") && !filtered.match(/^<[@#&]|(@&)\d+>$/)) {
+				return msg.delete();
+			}
+		}
+	});
+};
```

Comment:
Please have formatting be consistent with other files. (Tabs -> Spaces, 4 spaces per tab)